core/domain/voiceover_services.py

```diff
@@ -362,7 +362,7 @@ def get_voiceover_filenames(
     # method instead.
     # https://github.com/python/mypy/issues/9590
     k = lambda voiceover: voiceover['duration_secs']
-    contributed_voiceovers.sort(key=k)
+    contributed_voiceovers.sort(key=k, reverse=True)
```

Comment:
Here also I'm not sure this is needed. Why reversing the sorting order?

Reply:
Just a note, I can confirm that this is correct. (We're trying to assign accents to voiceovers, and the code is supposed to surface the 5 longest voiceovers rather than the 5 shortest.)